include/highfive/bits/H5Node_traits.hpp

```diff
@@ -110,7 +110,38 @@ class NodeTraits {
     ///
     /// \brief return the name of the object with the given index
     /// \return the name of the object
-    std::string getObjectName(size_t index) const;
+    std::string getName(size_t index) const;
```

Comment:
```suggestion
    std::string getObjectName(size_t index) const;
```
Otherwise you break backward compatibility